### DISCUSSION TITLE
Trying to get GetBlock() on MainNet fails

### DISCUSSION THREAD

**Discussion by xncv-hs** [bmresearch/Solnet#324] [Q&A]
When I try to use RpcClient.GetBlock(*any block*) on MainNet, I always receive the error: "Too many requests for a specific RPC call, contact your app developer or support@rpcpool.com."

The same query works fine on DevNet and TestNet.

**Comment by hoakbuilds** [comment]
You may be getting rate limited by RPCPool. I assume this is using the public RPCs using `Cluster.MainNet`? You should try using the GenesysGo public one `https://ssc-dao.genesysgo.net`

**Comment by airose007** [comment]
If you looking for help with building a sniper bot, join here https://t.me/+U_BCU0XDRYU0ZWQy
